src/app/home-page/recent-item-list/recent-item-list.component.html

```diff
@@ -0,0 +1,14 @@
+<ng-container *ngVar="(itemRD$ | async) as itemRD">
+        <div class="mt-4" *ngIf="itemRD?.hasSucceeded && itemRD?.payload?.page.length > 0" @fadeIn>
+                <div class="d-flex flex-row border-bottom mb-4 pb-4 ng-tns-c416-2"></div>
+                <h2>Recent Submissions </h2>
```

Comment:
Should be a `i18n` label; eg `{{'home.recent-submissions.head' | translate}}` & add that label to `en.json5`

Reply:
home.recent-submissions.head added into en.jon